api/v1/paths/developers/{developerId}.js

```diff
@@ -0,0 +1,27 @@
+const appRoot = require('app-root-path');
+
+const developersDao = require('../../db/oracledb/developers-dao');
+
+const { errorBuilder, errorHandler } = appRoot.require('errors/errors');
+const { openapi: { paths } } = appRoot.require('utils/load-openapi');
+
+/**
+ * @summary Get developer by unique ID
+ */
+const get = async (req, res) => {
+  try {
+    const { developerId } = req.params;
+    const result = await developersDao.getDeveloperById(developerId);
+    if (!result.data || !result.data.length) {
```

Comment:
I could be wrong, but can't this be shortened as `if (!result)`?

Reply:
You are right and you exposed a larger bug with 404 results that I fixed.